File: VisitBosnia/visit_bosnia_mobile/lib/model/post/post_insert_request.dart
import 'package:visit_bosnia_mobile/model/appUser/app_user.dart';

class PostInsertRequest {
  int? appUserId;
  int? forumId;
  String? createdTime;
  String? title;
  String? content;

  PostInsertRequest({
    this.appUserId,
    this.forumId,
    this.createdTime,
    this.title,
    this.content,
  });

  PostInsertRequest.fromJson(Map<String, dynamic> json) {
    appUserId = json['appUserId'];
    forumId = json['forumId'];
    createdTime = json['createdTime'];
    title = json['title'];
    content = json['content'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['appUserId'] = this.appUserId;
    data['forumId'] = this.forumId;
    data['createdTime'] = this.createdTime;
    data['title'] = this.title;
    data['content'] = this.content;
    return data;
  }
}

File: VisitBosnia/VisitBosnia.Model/Requests/PostInsertRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace VisitBosnia.Model.Requests
{
    public class PostInsertRequest
    {
        public int AppUserId { get; set; }
        public int ForumId { get; set; }
        public DateTime CreatedTime { get; set; }
        public string Title { get; set; } = null!;
        public string Content { get; set; } = null!;

    }
}
